src/main/steps/edge-case/subject-details/content.ts

```diff
@@ -18,7 +18,7 @@ export const form: FormContent = {
       type: 'text',
       classes: 'govuk-input',
       label: l => l.subjectFullNameLabel,
-      validator: input => isFieldFilledIn(input) || isFieldLetters(input),
+      validator: input => isFieldFilledIn(input) || isMarkDownLinkIncluded(input),
```

Comment:
Why are we allowing numbers in names?

Reply:
Per Jon's feedback: he agreed to supporting all characters users may need to enter, including numbers and symbols e.g. Jon Matthews the 2nd, lawyers 4 u